mobile/library/objective-c/EnvoyConfiguration.mm

```diff
@@ -310,29 +302,20 @@ - (nullable NSString *)resolveTemplate:(NSString *)templateYAML {
 - (BOOL)compareYAMLWithProtoBuilder:(NSString *)yaml {
   Envoy::Platform::EngineBuilder builder;
 
-  builder.addGrpcStatsDomain([self.grpcStatsDomain toCXXString]);
-  builder.addConnectTimeoutSeconds(self.connectTimeoutSeconds);
-  builder.addDnsRefreshSeconds(self.dnsRefreshSeconds);
-  builder.addDnsFailureRefreshSeconds(self.dnsFailureRefreshSecondsBase,
-                                      self.dnsFailureRefreshSecondsMax);
-  builder.addDnsQueryTimeoutSeconds(self.dnsQueryTimeoutSeconds);
-  builder.addDnsMinRefreshSeconds(self.dnsMinRefreshSeconds);
-  builder.enableDnsCache(self.enableDNSCache, self.dnsCacheSaveIntervalSeconds);
-  builder.addMaxConnectionsPerHost(self.maxConnectionsPerHost);
-  builder.addH2ConnectionKeepaliveIdleIntervalMilliseconds(
-      self.h2ConnectionKeepaliveIdleIntervalMilliseconds);
-  builder.addH2ConnectionKeepaliveTimeoutSeconds(self.h2ConnectionKeepaliveTimeoutSeconds);
-  builder.addStatsFlushSeconds(self.statsFlushSeconds);
-
-  builder.setAppVersion([self.appVersion toCXXString]);
-  builder.setAppId([self.appId toCXXString]);
-  builder.setDeviceOs("iOS");
+  for (EnvoyNativeFilterConfig *nativeFilterConfig in self.nativeFilterChain) {
```

Comment:
I aligned the order in which things are done here to match what's in `-[EnvoyConfiguration resolveTemplate:]`.